Mathlib/Topology/Order.lean

```diff
@@ -409,6 +414,11 @@ theorem coinduced_iSup {ι : Sort w} {t : ι → TopologicalSpace α} :
     (⨆ i, t i).coinduced f = ⨆ i, (t i).coinduced f :=
   (gc_coinduced_induced f).l_iSup
 
+@[simp]
+theorem coinduced_sSup {s : Set (TopologicalSpace β)} :
+    TopologicalSpace.coinduced g (sSup s) = sSup ((TopologicalSpace.coinduced g) '' s) := by
+  rw [sSup_eq_iSup', sSup_image', coinduced_iSup]
```

Comment:
Does this also work as `(gc_coinduced_induced f).l_sSup` or similar?

Reply:
Annoyingly, no:
```
type mismatch
  GaloisConnection.l_sSup (gc_coinduced_induced g)
has type
  TopologicalSpace.coinduced g (sSup s) = ⨆ a ∈ s, TopologicalSpace.coinduced g a : Prop
but is expected to have type
  TopologicalSpace.coinduced g (sSup s) = sSup (TopologicalSpace.coinduced g '' s) : Prop
```

Reply:
I think the issue here is that we can't agree whether `⨆ a ∈ s, f a` or `sSup (f '' s)` is the preferred spelling, and so the library lemma uses the first and yours uses the second. I think @urkud prefers the former, but I'd argue that having `sSup` on both sides of the equality is more important, so I prefer the latter.

Reply:
So are you arguing for `GaloisConnection.l_sSup'` with your preferred spelling? And should this be a separate PR? I am quite happy to dump this lemma completely in this PR, it's unused, it's only there for symmetry's sake.

Reply:
The Zulip discussion [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/normal.20form.20for.20pullback.20of.20Inf.20of.20topologies/near/472676441) seems to indicate that what I'm doing here is fine.